openapi.yaml

```diff
@@ -3778,15 +3796,21 @@ components:
           items:
             type: string
           type: array
-        jira:
-          $ref: '#/components/schemas/JIRAFilter'
         lines:
           description: Split by changed number of lines (inserted + removed).
           items:
             minimum: 0
             type: integer
           minLength: 2
           type: array
+        jira:
+          $ref: '#/components/schemas/JIRAFilter'
+        environments:
+          description: Calculate deployment metrics separately for each environment
```

Comment:
Hence the combobox. Indeed, averaging the metrics for all envs does not make sense.

Reply:
I remember that we discussed making this a setting near the release settings. But those settings do not satisfy the desire to quickly see how metrics change between several environments.